### PR TITLE
User friendly experience for 7 day trade restriction

### PR DESCRIPTION
**Pull request by jtormey**
Depends on
- [x] https://github.com/blockchain/bitcoin-coinify-client/pull/1
- [x] trade restriction added to / tested on Coinify staging

### CODE REVIEW

**Comment by Sjors** [comment]
It showed this error message when creating a new wallet:
new wallet

I was able to workaround that by adding a check for `exchange.hasAccount`.

However it seems that this error message always appears, even if Coinify doesn't set it. Not sure what to do about that...

File: app/partials/buy-sell.jade [hunk 1]
```diff
@@ -13,6 +13,7 @@
                         modal-open="status.modalOpen"
                         change-currency="changeCurrency(currency)"
                         limits="limits"
+                        can-trade="exchange.profile.canTrade"
```
Comment (Sjors) [inline app/partials/buy-sell.jade]: `exchange.profile.canTrade` is undefined (falsy) when the user doesn't have an account or when the `canTrade` field is missing. A "negatrive" attribute would be safer here, e.g. `trade-suspended-until`.

Comment (jtormey) [inline app/partials/buy-sell.jade]: Updated to use a negative check (now `trading-disabled`)

**Comment by jtormey** [comment]
Tested with 10-minute delay on staging. Works great.

**Pull request merged**